crates/factor-key-value/src/lib.rs

```diff
@@ -40,6 +41,7 @@ impl Factor for KeyValueFactor {
     fn init<T: Send + 'static>(&mut self, mut ctx: InitContext<T, Self>) -> anyhow::Result<()> {
         ctx.link_bindings(spin_world::v1::key_value::add_to_linker)?;
         ctx.link_bindings(spin_world::v2::key_value::add_to_linker)?;
+        ctx.link_bindings(spin_world::wasi::keyvalue::store::add_to_linker)?;
```

Comment:
Do we need to link the atomics and batch interfaces too?  In wasi-blobstore, I seemed to run into a problem if I didn't link the additional interfaces.

Reply:
added!